packages/checkout/widgets-lib/src/widgets/bridge/components/BridgeForm.tsx

```diff
@@ -15,7 +15,7 @@ import {
 import {
   useCallback, useContext, useEffect, useMemo, useRef, useState,
 } from 'react';
-import { BigNumber, utils } from 'ethers';
+import { BigNumber, ethers, utils } from 'ethers';
```

Comment:
you already have access to `BigNumber`

```suggestion
import { BigNumber, utils } from 'ethers';
```